### PR TITLE
feat(profiles): when editing profile- a user can save changes and the…

### LINKED ISSUE #490: Close edit profile section when save info in profile page sponsor
When editing profile – a user can save changes and the editor closes.

"Warning: save changes before closing." (dialog)
"Changes saved." (snackbar)

### PR DESCRIPTION
**Pull request by Edgar097**
Resolves #490 

image

-----------

image

### CODE REVIEW

File: webapp/src/routes/EditProfile/EditProfileSponsor.js [hunk 1]
```diff
@@ -638,14 +638,17 @@ const EditProfileSponsor = ({ profile, isCompleting, onSubmit, loading }) => {
...
         <Button
           className={classes.saveBtn}
           variant="contained"
           color="secondary"
           onClick={() => prepareDataForSubmitting()}
         >
+
           {t('common.save')}
         </Button>
+
```
Comment (JustinCast) [inline webapp/src/routes/EditProfile/EditProfileSponsor.js:651]: Are these blank spaces necessary?

File: webapp/src/routes/EditProfile/EditProfileSponsor.js [hunk 2]
```diff
@@ -638,12 +638,14 @@ const EditProfileSponsor = ({ profile, isCompleting, onSubmit, loading }) => {
             {t('common.cancel')}
           </Button>
         </Link>
+
```
Comment (JustinCast) [inline webapp/src/routes/EditProfile/EditProfileSponsor.js:641]: Are these spaces necessary ?

File: webapp/src/routes/EditProfile/EditProfileSponsor.js [hunk 3]
```diff
@@ -638,12 +638,14 @@ const EditProfileSponsor = ({ profile, isCompleting, onSubmit, loading }) => {
...
         <Button
           className={classes.saveBtn}
           variant="contained"
           color="secondary"
           onClick={() => prepareDataForSubmitting()}
         >
+
```
Comment (JustinCast) [inline webapp/src/routes/EditProfile/EditProfileSponsor.js:648]: Same here

**Review by JustinCast — APPROVED** [review]
LGTM

**Review by steph222 — APPROVED** [review]

**Review by AngeloCG97 — APPROVED** [review]

**Review by tetogomez — APPROVED** [review]
LGTM

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**